src/components/BillCategoryTag/BillCategoryTag.story.svelte

```diff
@@ -7,8 +7,8 @@
 </script>
 
 <Hst.Story title="BillCategoryTag" layout={{ type: 'grid' }}>
-	<Hst.Variant title="BillCategory">
-		<BillCategoryTag {isLarge} />
+	<Hst.Variant title="PublicTransport">
```

Comment:
This should be the last 2 feedbacks. Because we have only one variant you don't need to put it in the `Hst.Variant`, just put it in the `Hst.Story` and remove the `layout` attribute.